cmd/server/assets/codes/bulk-issue.html

```diff
@@ -245,8 +269,15 @@
               flash.alert("Successfully issued " + total + " codes.");
             }
           }
+
+          if (totalErrs > 0) {
+            flash.error("Received errors for " + totalErrs + " entries. See error table for details.");
```

Comment:
Do we need to `return` here? Might not need to but it's hard to tell with the collapsed diff.

Reply:
nah the rest just resets the div

Reply:
/unhold